.circleci/config.yml

```diff
@@ -0,0 +1,235 @@
+version: 2
+references:
+  build_containers_config: &build_container_config
+    working_directory: ~/repo
+    docker:
+      - image: circleci/ruby:2.5.3-node-browsers
```

Comment:
possibly could use circleci/ruby:2.5.3-node-browsers-legacy to include phantomJS
https://circleci.com/docs/2.0/browser-testing/